kerastuner/engine/hyperparameters.py

```diff
@@ -558,44 +558,61 @@ def register(self, name, type, config):
         self.values[full_name] = value
         return value
 
-    def get(self, name):
+    def get(self, name, default=None):
```

Comment:
Hyperparameters already have a default value that is defined at instantiation time. Wouldn't it be dangerous or confusing to expose a different way of specifying a default value, that overrides the originally-specified default? It seems safer to have `get` always error out in case the name is not found.

Reply:
Done. True, removed